zcash_client_backend/src/data_api/chain.rs

```diff
@@ -115,28 +208,43 @@ where
 /// use zcash_client_sqlite::{
 ///     BlockDB,
 ///     WalletDB,
+///     error::SqliteClientError,
+///     wallet::init::init_wallet_db,
 /// };
 ///
+/// # // doctests have a problem with sqlite IO, so we ignore errors
+/// # // generated in this example code as it's not really testing anything
+/// # fn main() {
+/// #   test();
+/// # }
```

Comment:
Ah, I assume this is why you also don't return an error from `fn main()` at my earlier comment.

Reply:
Added a comment.